shotover-proxy/src/server.rs

```diff
@@ -182,6 +206,7 @@ impl<C: Codec + 'static> TcpCodecListener<C> {
                 .unwrap_or_else(|_| "Unknown peer".to_string());
 
             // Create the necessary per-connection handler state.
+            debug!("New connection from {}", &conn_string);
```

Comment:
I think this new connection log came from a merge conflict? I moved the new connection log down to line 240 in the span PR.